### DISCUSSION TITLE
What am I supposed to see when I visit my domain after installing dokku?

### DISCUSSION THREAD

**Discussion by NathanWailes** [dokku/dokku#7095] [Q&A] [answered]
I went through [this guide][1] but I'm not sure how to tell if I installed dokku correctly. When I go to my whatdo.com domain I just see the "Welcome to nginx" page that I saw before I installed dokku. When I go to https://ruby-getting-started.whatdo.com/ I get no response at all.

[1]: https://dokku.com/docs/getting-started/install/digitalocean/

**Comment by dewey** [comment, marked as answer]
Hey, you'll have to give a bit more information than that. This guide just installs Dokku, if you want to host a website you'll have to create a small static site and push to it. The steps are somewhat like the following:

1) Make sure your domain's DNS records are pointed at your server
2) dokku apps:create example.com
3) dokku domains:set example.com example.com www.example.com
4) dokku letsencrypt:enable example.com
5) Create a git repository with a index.html and a .static file in there
6) git remote add dokku dokku@example.com:example.com
7) git push
8) Your website should now be reachable

**Reply by NathanWailes** [reply]
Thank you for responding. I ended up getting everything working. My question here was just about what a person should expect to see when visiting a Digital Ocean VPS IP address after having installed Dokku but possibly before having installed a particular app. At the moment my understanding is that there isn't some default Dokku landing page like what Nginx and Apache have, so a person should still expect to see the default Nginx landing page (since Dokku seems to use Nginx) when visiting their droplet's IP address.

**Reply by dewey** [reply]
Glad it's working now! I don't think there's a default landing page, but I think it would be a good idea to have a friendly "Welcome to Dokku, run these 4 commands to show your first site here" landing page.
